pkg/gather/collect/collect.go

```diff
@@ -106,6 +120,30 @@ func NewCollector(
 		logsLimitBytes:     logsLimitBytes,
 		collectedResources: apimachineryutilsets.Set[string]{},
 	}
+
+	c.podRuntimeCollectors = []PodRuntimeCollector{
+		{
+			Collect: c.collectPodLogs,
+		},
+		{
+			Filter:  controllerhelpers.IsScyllaContainerRunning,
+			Collect: c.collectContainerCommandOutputFunc(naming.ScyllaContainerName, "nodetool-status.log", []string{"nodetool", "status"}),
+		},
+		{
+			Filter:  controllerhelpers.IsScyllaContainerRunning,
+			Collect: c.collectContainerCommandOutputFunc(naming.ScyllaContainerName, "nodetool-gossipinfo.log", []string{"nodetool", "gossipinfo"}),
+		},
+		{
+			Filter:  controllerhelpers.IsScyllaContainerRunning,
+			Collect: c.collectContainerCommandOutputFunc(naming.ScyllaContainerName, "df.log", []string{"df", "-h"}),
+		},
+		{
+			Filter:  controllerhelpers.IsNodeConfigPod,
+			Collect: c.collectContainerCommandOutputFunc(naming.NodeConfigAppName, "kubelet-cpu_manager_state.log", []string{"cat", "/host/var/lib/kubelet/cpu_manager_state"}),
+		},
+	}
```

Comment:
Just food for thought: maybe we should start extracting collectors for specific resources to dedicated structs & files so we have better separation of concerns? `Collector` implementation grows, and it's becoming hard to read from my perspective as it touches various resources where some of them have special handling (like Pods).

I think that for readability's and segregation's sake, we could define these runtime collectors inline in `collectPodRuntimeInformation` for now (to avoid refactoring), so they do not leak to the top-level logic (which the `Collector` constructor is from my perspective).